packages/fluentui/react-northstar/src/components/Dialog/utils/index.tsx

```diff
@@ -0,0 +1,17 @@
+const _disableIosBodyScroll = (event: TouchEvent) => {
+  event.preventDefault();
+};
+
+export function enableBodyScroll(target: HTMLElement): void {
+  if (target) {
+    target.style.overflow = 'unset';
+    target.removeEventListener('touchmove', _disableIosBodyScroll);
+  }
+}
+
+export function disableBodyScroll(target: HTMLElement): void {
+  if (target) {
+    target.style.overflow = 'hidden';
+    target.addEventListener('touchmove', _disableIosBodyScroll, { passive: false, capture: false });
```

Comment:
FYI: IE11 :( However, it will not throw, so we are fine

https://caniuse.com/#feat=passive-event-listener
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Browser_compatibility

Reply:
Alright